File: leetCode/TwoSum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 * 
 * https://leetcode.com/problems/two-sum/
 */
var twoSum = function(nums, target) {
    let output = new Array(2)
    
    for(let i=0; i<nums.length; i++){
        for(let j=i+1; j<nums.length; j++){
            let sumOfTarget = nums[i] + nums[j]
            
            if(target === sumOfTarget){
                output[0] = i
                output[1] = j
                
                return output
            }
        }
    }
    return -1
};

File: leetCode/TwoSum.java
//https://leetcode.com/problems/two-sum/

class Solution {
    public int[] twoSum(int[] nums, int target) {
        
        int[] output = new int[2];
        
        for(int i=0; i<nums.length; i++){
            for(int j=i+1; j<nums.length; j++){
                int sumOfNums = nums[i] + nums[j];
                
                if(sumOfNums == target){
                    output[0] = i;
                    output[1] = j;
                    
                    return output;
                }
            }
        }     
        return output;      
    }
}